deploy/crds/zookeeper.pravega.io_zookeeperclusters_crd.yaml

```diff
@@ -3659,6 +3659,9 @@ spec:
                   will be using It can take either Ephemeral or persistence Default
                   StorageType is Persistence storage
                 type: string
+              triggerRollingRestart:
```

Comment:
would it make sense to have a `revision` that can be an incremental version/sequence instead of a boolean switch?
This way the users can update it with a deployment timestamp if they want to kick a rolling restart.

Reply:
if we change this to a revision number, then we will also need to support a rollback to a previous version. in this case, we will need to store the previous crd versions in order to be able to rollback to it. this is very similar to the functionality already provided by helm charts.
the deployment timestamp is already captured in the "restartTime" annotation which is being added to the pods so perhaps it would not be needed to capture it here as well. what do you think?